tools/integration_tests/implicitdir/implicitdir_test.go

```diff
@@ -147,6 +147,31 @@ func createTempFile() string {
 	return fileName
 }
 
+func executeTest(flags []string, m *testing.M) (successCode int) {
+	var err error
+	for i := 0; i < len(flags); i++ {
+		if err = mountGcsfuse(flags[i]); err != nil {
```

Comment:
couple of questions here:
1. Does it log the mount command or someother information which will help us identify the test failure related to which mount?
2. successCode = m.Run() , not acting on the success code here. does it mean, if tests failed for a specific mount we are proceeding with test execution for other mounts?

Reply:
1. In logFIle I added mounting command and then It will run test related to that mount.
2. If any test will fail it will show in terminal only. successcode is only for os.Exit() at the end of all the test case executed.